pymtl3/passes/sverilog/import_/verilator_wrapper.c.template

```diff
@@ -163,3 +164,14 @@ void eval( V{component_name}_t * m ) {{
   #endif
 
 }}
+
+//------------------------------------------------------------------------
+// assert_en()
+//------------------------------------------------------------------------
+// Enable or disable assertions controlled by --assert
+
```

Comment:
if i remember correctly, the assertion error in C land will turn into segmentation fault that python land cannot catch and directly terminate the simulation.

Reply:
It's possible to overwrite Verilator's `stop` function and swap in some code that calls Python C API to raise a Python exception. We can find out later.

Reply:
hmmm not sure if Python C API is restricted to only CPython. Anyways, maybe you could add some TODO to record  this conversation.